Code/cmake/Modules/FindCairo.cmake

```diff
@@ -42,17 +42,28 @@ PKG_CHECK_MODULES(PC_CAIRO cairo) # FIXME: After we require CMake 2.8.2 we can p
 
 FIND_PATH(CAIRO_INCLUDE_DIRS
     NAMES cairo.h
-    HINTS ${PC_CAIRO_INCLUDEDIR}
+    HINTS ${CAIRO_INCLUDE_DIR}
+          ${PC_CAIRO_INCLUDEDIR}
           ${PC_CAIRO_INCLUDE_DIRS}
     PATH_SUFFIXES cairo
 )
 
 FIND_LIBRARY(CAIRO_LIBRARIES
     NAMES cairo
-    HINTS ${PC_CAIRO_LIBDIR}
+    HINTS ${CAIRO_LIBRARY_DIR}
+          ${PC_CAIRO_LIBDIR}
           ${PC_CAIRO_LIBRARY_DIRS}
 )
 
+IF (UNIX AND NOT APPLE)
+	PKG_CHECK_MODULES(PC_XAU xau) # FIXME: After we require CMake 2.8.2 we can pass QUIET to this call.
```

Comment:
We already require cmake 3.5, so we should either add `QUIET` or remove this comment

Reply:
@greglandrum done.